src/parser/file_utils.c

```diff
@@ -16,14 +20,11 @@ void	open_infile(const char *file, int *in_file)
 	}
 }
```

Comment:
0644 can be the file_rights mode.

Reply:
Not sure what you mean with this one? 
I replaced 0644 with a macro defined as `#define FILE_RIGHTS 0664`